tests/conftest.py

```diff
@@ -175,21 +184,25 @@ def expected_catalog_from_db():
                      'inclusion': 'unsupported',
                      'description': 'Unsupported column type unknown'}},
              'type': 'object'},
-         'is_view': True,
          'stream': 'view1',
          'metadata': [
              {'breadcrumb': (),
               'metadata': {'selected-by-default': False,
                            'forced-replication-method': {
                             'replication-method': 'FULL_TABLE',
                             'reason': 'No replication keys found from table'
-                           }}},
+                           },
+                           'table-key-properties': [],
+                           'is-view': True,
```

Comment:
Should this include `view-key-properties`, given that `is-view` is `True`?

Reply:
Yeah, that sounds right.